test/soca/CMakeLists.txt

```diff
@@ -29,6 +29,25 @@ endforeach(FILENAME)
 # tests below require the full build
 if(BUILD_GDASBUNDLE)
 
+  # link input file from iodaconv to test directory
+  #file(CREATE_LINK ${PROJECT_SOURCE_DIR}/iodaconv/test/testinput/dbuoy_20180415_bufr ${PROJECT_BINARY_DIR}/test/testdata/dbuoy_20180415_bufr SYMBOLIC)
```

Comment:
remove commented out line

Reply:
This change still needs addressed @ShastriPaturi

Reply:
Done